event_project/models/event_event.py

```diff
@@ -1,72 +1,87 @@
-# -*- encoding: utf-8 -*-
-##############################################################################
-#
-#    OpenERP, Open Source Management Solution
-#    This module copyright :
-#        (c) 2014 Antiun Ingenieria, SL (Madrid, Spain, http://www.antiun.com)
-#                 Endika Iglesias <endikaig@antiun.com>
-#                 Antonio Espinosa <antonioea@antiun.com>
-#                 Javier Iniesta <javieria@antiun.com>
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import models, fields, api
+from odoo import api, fields, models
 
 
 class EventEvent(models.Model):
     _inherit = 'event.event'
 
-    project_template = fields.Many2one(
-        comodel_name='project.project', string='Template project',
-        domain="[('state', '=', 'template')]")
-    project = fields.Many2one(
+    project_id = fields.Many2one(
         comodel_name='project.project', string='Related project',
-        readonly=True)
+        help="Project end date will be updated with event start date.")
+    task_ids = fields.One2many(
```

Comment:
A related `One2many` without `readonly`??!! I don't think so... This for sure is causing troubles.